pkg/services/ngalert/metrics/historian.go

```diff
@@ -7,17 +7,59 @@ import (
 )
 
 type Historian struct {
-	WriteDuration *instrument.HistogramCollector
+	Info              *prometheus.GaugeVec
+	TransitionsTotal  *prometheus.CounterVec
+	TransitionsFailed *prometheus.CounterVec
+	WritesTotal       *prometheus.CounterVec
+	WritesFailed      *prometheus.CounterVec
+	WriteDuration     *instrument.HistogramCollector
+	BytesWritten      prometheus.Counter
 }
 
 func NewHistorianMetrics(r prometheus.Registerer) *Historian {
 	return &Historian{
+		Info: promauto.With(r).NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "state_history_info",
+			Help:      "Information about the state history backend.",
+		}, []string{"backend"}),
+		TransitionsTotal: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "state_history_transitions_total",
+			Help:      "The total number of state transitions processed by the state historian.",
```

Comment:
```suggestion
			Help:      "The total number of state transitions processed.",
```

Reply:
"state historian" is a system component  (there's also no mention in any other metrics) I'd remove it.